niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -811,7 +849,7 @@ public void writeToParcel(Parcel parcel, int i) {
         // In P this permission only limited the accounts returned from APIs, but starting in Q
         // other APIs under the UserManager were guarded by this permission.
         mPermissionTasks.put(permission.GET_ACCOUNTS_PRIVILEGED,
-                new PermissionTest(false, Build.VERSION_CODES.Q, () -> {
+                new PermissionTest(false, Build.VERSION_CODES.P, () -> {
```

Comment:
Why was this changed to P? I thought when run against a device running Android P, it did not throw a `SecurityException` when the permission was not granted.

Reply:
Because this GET_ACCOUNTS_PRIVILEGED permission is avialable before ANDROID P.
I think we should not append exceptional meaning to this parameter to maintain the consistency of this project.
(and the change will not break the test.)

Reply:
Sounds good, I was just concerned that the test would fail on devices running Android P when the permission is not granted, but if it works in that case, then this looks good to me.